src/asami/query.cljc

```diff
@@ -638,11 +638,33 @@
       ;; calculate the aggregates from the final results and project
       (aggregate-over find inner-results))))
 
+(s/defn rewrite-wildcards
```

Comment:
My bad with `query-entry` below. This function does not need schema, especially given the destructuring I mention immediately below